sellotape/sellotape_dj/main_app/admin.py

```diff
@@ -1,3 +1,27 @@
 from django.contrib import admin
+from django.contrib.auth.models import User
 
 # Register your models here.
+
+# https://docs.djangoproject.com/en/dev/topics/auth/customizing/#extending-the-existing-user-model
+class User(models.Model):
+	# Inherits Attributes from User default instance
+    user 	= models.OneToOneField(User, on_delete=models.CASCADE)
+    # Generates a user ID which is the primary identifier to the user
+    user_id	= models.AutoField(primary_key=True)
+    # User avatar, optional: limit the image instance height and width
+    avatar	= models.ImageField()
+    country	= models.CharField(default='Israel')
+    
+	class city(models.TextChoices):
```

Comment:
Class names in Python usually start with a capital letter.

This line aught to be at the same indention level as the fields above. Please try to keep orderly indention - this is 
especially important with Python

With regards to the choices below, consider making the key part of the choice an int for more efficient DB storage and lookup. Also - isn't it a bit too early to try to introduce a language translation library? If you do want to use it - you probably need to import the `_()` function from somewhere.